ci-operator/step-registry/platformoperator/valid/amq-streams/platformoperator-valid-amq-streams-commands.sh

```diff
@@ -6,11 +6,11 @@ cat <<EOF > ${SHARED_DIR}/manifest_cert_manager.yaml
 apiVersion: platform.openshift.io/v1alpha1
 kind: PlatformOperator
 metadata:
-  name: cert-manager-po
+  name: amq-streams-po
 spec:
   package:
-    name: openshift-cert-manager-operator
+    name: amq-streams
```

Comment:
@jianzhangbjz `amq-streams` package is used by the case of logging team and netobserve team.
it possible cause conflicts when running case.

suggest to change to other operators.